### PR TITLE
Updated readme for istio.

### PR DESCRIPTION
**Pull request by gyliu513**
**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by linsun** [comment]
@andraxylia @ldemailly @gyliu513 could you explain why we need to have istio-one-namespace.yaml and istio.yaml?  I would like to minimize choices for the users if we could.

**Comment by gyliu513** [comment]
Thanks @linsun for pointing this out, when I create this PR, I also have same concern as you but do not clear about the history of those two files. 

@andraxylia @ldemailly can you comment? Thanks!

**Comment by andraxylia** [comment]
We will not include  istio-one-namespace.yaml in the release, this is just for the e2e tests in shared cluster. It is similar to what istio 0.1 provided, all in one namespace.

File: install/kubernetes/README.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+Please follow the installation instructions from [istio.io](https://istio.io/docs/setup/kubernetes/quick-start.html).
 
 ## Directory structure
 This directory contains files needed for installing Istio on a Kubernetes cluster.
 
 * istio.yaml - use this file for installation without authentication enabled
 * istio-auth.yaml - use this file for installation with authentication enabled
-* istio-cluster-wide.yaml - use this file for installation cluster-wide with authentication enabled
+* istio-one-namespace.yaml - use this file for installation without authentication enabled and CA only watch one namespace
+* istio-one-namespace-auth.yaml - use this file for installation without authentication enabled and CA only watch one nemespace
```
Comment (andraxylia) [inline install/kubernetes/README.md]: use this for restricting istio and the applications to a single namespace.

File: install/kubernetes/README.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+* istio-initializer.yaml - use this file for installation of istio initializer for transparent injection.
```
Comment (andraxylia) [inline install/kubernetes/README.md]: move this above, right after istio-auth.

File: install/kubernetes/README.md (hunk 1 above)
Comment (andraxylia) [reply install/kubernetes/README.md]: sandbox, used for testing etc.

File: install/kubernetes/README.md [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
-Please follow the installation instructions from [istio.io](https://istio.io/docs/tasks/installing-istio.html).
+Please follow the installation instructions from [istio.io](https://istio.io/docs/setup/kubernetes/quick-start.html).
 
 ## Directory structure
 This directory contains files needed for installing Istio on a Kubernetes cluster.
 
 * istio.yaml - use this file for installation without authentication enabled
 * istio-auth.yaml - use this file for installation with authentication enabled
-* istio-cluster-wide.yaml - use this file for installation cluster-wide with authentication enabled
+* istio-one-namespace.yaml - use this file for installation without authentication enabled and CA only watch one namespace
```
Comment (ldemailly) [inline install/kubernetes/README.md]: put a period after CA and start a new sentence "Istio control plane and app in the same watched namespace."

Comment (gyliu513) [reply install/kubernetes/README.md]: Done

File: install/kubernetes/README.md (hunk 1 above)
Comment (gyliu513) [reply install/kubernetes/README.md]: Done

File: install/kubernetes/README.md (hunk 2 above)
Comment (gyliu513) [reply install/kubernetes/README.md]: Done

File: install/kubernetes/README.md [hunk 4]
```diff
@@ -1,13 +1,15 @@
...
 
 ## Directory structure
 This directory contains files needed for installing Istio on a Kubernetes cluster.
 
 * istio.yaml - use this file for installation without authentication enabled
 * istio-auth.yaml - use this file for installation with authentication enabled
-* istio-cluster-wide.yaml - use this file for installation cluster-wide with authentication enabled
+* istio-initializer.yaml - use this file for installation of istio initializer for transparent injection.
+* istio-one-namespace.yaml - use this file for installation without authentication enabled and CA. Istio control plane and applications will be in one single namespace, mainly used for testing.
+* istio-one-namespace-auth.yaml - use this file for installation without authentication enabled and CA. Istio control plane and applications will be in one single namespace, mainly used for testing.
```
Comment (andraxylia) [inline install/kubernetes/README.md]: replace without with with.

Comment (andraxylia) [reply install/kubernetes/README.md]: remove CA reference from both. CA is always installed.

Comment (gyliu513) [reply install/kubernetes/README.md]: Thanks @andraxylia , it is a bit late to update as it was already merged, created a new PR here https://github.com/istio/istio/pull/960

**Pull request merged**